src/assets/components/Tile.jsx

```diff
@@ -0,0 +1,10 @@
+function Tile({ title, description, imageUrl, imgAlt}) {
+    return (
+        <section>
+            <h2>{title}</h2>
+            <p>{description}</p>
+            <img src={imageUrl} alt={imgAlt} />
```

Comment:
conditie voor image of text is in de Tile component meegegeven als volgt:
       <section>
            {image && <img src={image} alt={imgDescription} />}
            <h2>{title}</h2>
            {children}
        </section>